docs/source/core_services/multiplatform/Multiplatform-Communication.rst

```diff
@@ -0,0 +1,115 @@
+ .. _MultiPlatform-Communication:
+===========================
+MultiPlatform Communication
+===========================
+
+To connect to remote VOLTTRON platforms, we would need platform discovery information of the remote platforms. This
+information contains the platform name, VIP address and serverkey of the remote platforms and we need to provide this
+as part of Multiplatform configuration.
+
+Configuration
+*************
+
+To ensure a secure connection between connecting platforms, each connection need to be authenticated and authorized.
+
+Setup Mode For Automatic Authorization
+**************************************
+*Note: It is necessary for each platform to have a web server if running in setup mode *
```

Comment:
Instead of using a *  put it in a ..Note section....that will highlight it in a blue box...see https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#colored-boxes-note-seealso-todo-and-warnings for examples.

Reply:
This one is very important that people don't gloss over this fact.